variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "env_code" {}
+variable "vpc_cidr" {}
+variable "public_cidr" {}
+variable "private_cidr" {}
```

Comment:
could you please, add newline at the end of file

Reply:
Done